### PR TITLE
Add Waiting component and tests - 1 failing - CODE REVIEW PLZ

### PR DESCRIPTION
**Pull request by natdudley-EDA**

### CODE REVIEW

**Comment by H-THO** [comment]
I made this pull request actually so FYI Nat will be providing review as well 🐶

**Comment by brendan-olphert** [comment]
Thanks Hywel. looks very much on the right track, we can have a look at the test-id issue. thanks for creating the test-utils.js.

File: client/components/Waiting.test.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { renderWithRedux } from '../tests/test-utils'
+
```
Comment (natdudley-EDA) [inline client/components/Waiting.test.jsx]:
```suggestion
```

remove extra lines

File: client/components/Waiting.test.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (natdudley-EDA) [inline client/components/Waiting.test.jsx]:
```suggestion
```
remove extra lines

**Comment by H-THO** [comment]
Thanks all this looks great! 🌶️🌶️🌶️

**Comment by natdudley-EDA** [comment]
@H-THO looks good to me, but let's align some terminology.

Let's change:

- pending in actions and reducers to waiting to match
- use request and received instead of pending and success in inSeasonFoods actions creator.

**Comment by lorenzo-joseph-fernandez-19** [comment]
@H-THO  tests are passing. Good job there. Merging pull request

**Pull request merged**